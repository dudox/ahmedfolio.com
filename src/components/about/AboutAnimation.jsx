import React from "react";
import Social from "../Social";
import Testimonials from "../testimonial/TestimonialAnimation";
import Services from "../service/ServiceAnimation";
import Awards from "../award/AwardsAnimation";

const About = () => {
  return (
    <>
      <section id="about" className="section theme-light dark-bg">
        <div className="container">
          <div className="row align-items-center justify-content-center">
            <div
              className="col-md-6 col-lg-4"
              data-aos="fade-up"
              data-aos-duration="1200"
            >
              <div className="about-me">
                <div className="img">
                  <div className="img-in">
                    <img src="img/about/about-me.jpg" alt="" />
                  </div>

                  <Social />

                  {/* End social icon */}
                </div>
                {/* End img */}
                <div className="info">
                  <p>Software Engineer</p>
                  <h3>Ahmed Ahmed</h3>
                </div>
                {/* End info */}
              </div>
              {/* End about-me */}
            </div>
            {/* End col */}

            <div
              className="col-lg-7 ml-auto"
              data-aos="fade-up"
              data-aos-duration="1200"
              data-aos-delay="200"
            >
              <div className="about-info">
                <div className="title">
                  <h3>Biography</h3>
                </div>
                <div className="about-text">
                  <p>
                    I'm a Software Engineer with over 5 years of professional working
                    experience from Lagos, Nigeria. I find business models to be one of the apt. curious areas in development as it gives you an insight into the mind of the product owner and an avenue to creating unique problem solving solutions.
                  </p>
                  <p>
                    I find solace in what I do, brings me closer to understanding the world at large, problems people face on a daily bases and I get to work with the most brilliant and inspiring people on the planet. I am the type of person who pours water into a jar till it overflows because I love to give 110% at all time.
                  </p>
                </div>
                <div className="info-list">
                  <div className="row">
                    <div className="col-sm-6">
                      <ul>
                        <li>
                          <label>Name: </label>
                          <span>Ahmed Ahmed</span>
                        </li>
                        <li>
                          <label>Courses: </label>
                          <span>#MentalHealth, #ClimateChange</span>
                        </li>
                        <li>
                          <label>Interest: </label>
                          <span>AI, ioT, Cyber Security</span>
                        </li>
                        <li>
                          <label>Address: </label>
                          <span>Lagos, Nigeria</span>
                        </li>
                      </ul>
                    </div>
                    <div className="col-sm-6">
                      <ul>
                        <li>
                          <label>Phone: </label>
                          <span>(+234) 701 772 3208</span>
                        </li>
                        <li>
                          <label>Email: </label>
                          <span>work@ahmedfolio.com</span>
                        </li>
                        <li>
                          <label>Skype: </label>
                          <span>trybemark_1</span>
                        </li>
                        <li>
                          <label>Freelance: </label>
                          <span>Available</span>
                        </li>
                      </ul>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            {/* End col */}
          </div>

          {/* separated */}

          <div
            className="separated"
            style={{
              backgroundImage: `url(${
                process.env.PUBLIC_URL + "img/border-dark.png"
              })`,
            }}
          ></div>

          {/* End separated */}
          <div className="title">
            <h3>What I do?</h3>
          </div>

          <Services />

          {/* End .row */}

          {/* separated */}
          <div
            className="separated"
            style={{
              backgroundImage: `url(${
                process.env.PUBLIC_URL + "img/border-dark.png"
              })`,
            }}
          ></div>
          {/* End separated */}

          <div className="title">
            <h3>Awards.</h3>
          </div>

          <Awards />
          {/* End Awards */}

          {/* separated */}
          <div
            className="separated"
            style={{
              backgroundImage: `url(${
                process.env.PUBLIC_URL + "img/border-dark.png"
              })`,
            }}
          ></div>
          {/* End separated */}

          <div className="title">
            <h3>Testimonials.</h3>
          </div>

          <Testimonials />
          {/* End Testimonaial */}
        </div>
      </section>
    </>
  );
};

export default About;
