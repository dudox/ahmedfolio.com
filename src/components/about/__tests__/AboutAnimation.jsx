import React from "react";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import AboutAnimation from "../AboutAnimation";


it("renders", () => {
  const { asFragment } = render(<AboutAnimation  />);
  expect(asFragment()).toMatchSnapshot();
});