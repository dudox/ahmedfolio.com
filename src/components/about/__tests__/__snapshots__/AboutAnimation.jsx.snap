// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <section
    class="section theme-light dark-bg"
    id="about"
  >
    <div
      class="container"
    >
      <div
        class="row align-items-center justify-content-center"
      >
        <div
          class="col-md-6 col-lg-4"
          data-aos="fade-up"
          data-aos-duration="1200"
        >
          <div
            class="about-me"
          >
            <div
              class="img"
            >
              <div
                class="img-in"
              >
                <img
                  alt=""
                  src="img/about/about-me.jpg"
                />
              </div>
              <div
                class="nav social-icons justify-content-center"
              >
                <a
                  href="mailto:work@ahmedfolio.com"
                  rel="noreferrer"
                  target="_blank"
                >
                  <svg
                    class="feather feather-mail "
                    fill="none"
                    height="20"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="20"
                  >
                    <g>
                      <path
                        d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"
                      />
                      <polyline
                        points="22,6 12,13 2,6"
                      />
                    </g>
                  </svg>
                </a>
                <a
                  href="tel:+2347017723208"
                  rel="noreferrer"
                  target="_blank"
                >
                  <svg
                    class="feather feather-phone "
                    fill="none"
                    height="20"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="20"
                  >
                    <g>
                      <path
                        d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                      />
                    </g>
                  </svg>
                </a>
                <a
                  href="https://twitter.com/meedjnr"
                  rel="noreferrer"
                  target="_blank"
                >
                  <svg
                    class="feather feather-twitter "
                    fill="none"
                    height="20"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="20"
                  >
                    <g>
                      <path
                        d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"
                      />
                    </g>
                  </svg>
                </a>
                <a
                  href="https://www.linkedin.com/in/ahmed-ahmed-3a366016b"
                  rel="noreferrer"
                  target="_blank"
                >
                  <svg
                    class="feather feather-linkedin "
                    fill="none"
                    height="20"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="20"
                  >
                    <g>
                      <path
                        d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z"
                      />
                      <rect
                        height="12"
                        width="4"
                        x="2"
                        y="9"
                      />
                      <circle
                        cx="4"
                        cy="4"
                        r="2"
                      />
                    </g>
                  </svg>
                </a>
                <a
                  href="https://www.github.com/dudox"
                  rel="noreferrer"
                  target="_blank"
                >
                  <svg
                    class="feather feather-github "
                    fill="none"
                    height="20"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="20"
                  >
                    <g>
                      <path
                        d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
                      />
                    </g>
                  </svg>
                </a>
              </div>
            </div>
            <div
              class="info"
            >
              <p>
                Software Engineer
              </p>
              <h3>
                Ahmed Ahmed
              </h3>
            </div>
          </div>
        </div>
        <div
          class="col-lg-7 ml-auto"
          data-aos="fade-up"
          data-aos-delay="200"
          data-aos-duration="1200"
        >
          <div
            class="about-info"
          >
            <div
              class="title"
            >
              <h3>
                Biography
              </h3>
            </div>
            <div
              class="about-text"
            >
              <p>
                I'm a Software Engineer with over 5 years of professional working experience from Lagos, Nigeria. I find business models to be one of the apt. curious areas in development as it gives you an insight into the mind of the product owner and an avenue to creating unique problem solving solutions.
              </p>
              <p>
                I find solace in what I do, brings me closer to understanding the world at large, problems people face on a daily basis and I get to work with the most brilliant and inspiring people on the planet. I am the type of person who pours water into a jar till it overflows because I love to give 110% at all time.
              </p>
            </div>
            <div
              class="info-list"
            >
              <div
                class="row"
              >
                <div
                  class="col-sm-6"
                >
                  <ul>
                    <li>
                      <label>
                        Name: 
                      </label>
                      <span>
                        Ahmed Ahmed
                      </span>
                    </li>
                    <li>
                      <label>
                        Cause: 
                      </label>
                      <span>
                        #MentalHealth, #ClimateChange
                      </span>
                    </li>
                    <li>
                      <label>
                        Interest: 
                      </label>
                      <span>
                        AI, ioT, Cyber Security
                      </span>
                    </li>
                    <li>
                      <label>
                        Address: 
                      </label>
                      <span>
                        Lagos, Nigeria
                      </span>
                    </li>
                  </ul>
                </div>
                <div
                  class="col-sm-6"
                >
                  <ul>
                    <li>
                      <label>
                        Phone: 
                      </label>
                      <span>
                        (+234) 701 772 3208
                      </span>
                    </li>
                    <li>
                      <label>
                        Email: 
                      </label>
                      <span>
                        work@ahmedfolio.com
                      </span>
                    </li>
                    <li>
                      <label>
                        Skype: 
                      </label>
                      <span>
                        trybemark_1
                      </span>
                    </li>
                    <li>
                      <label>
                        Freelance: 
                      </label>
                      <span>
                        Available
                      </span>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="separated"
        style="background-image: url(img/border-dark.png);"
      />
      <div
        class="title"
      >
        <h3>
          What I do?
        </h3>
      </div>
      <div
        class="row"
      >
        <div
          class="col-md-6 col-lg-4 my-3"
          data-aos="fade-right"
          data-aos-delay=""
          data-aos-duration="1200"
        >
          <div
            class="feature-box-01"
          >
            <div
              class="icon"
            >
              <i
                class="icon icon-mobile"
              />
            </div>
            <div
              class="feature-content"
            >
              <h5>
                Web & App Development.
              </h5>
              <p>
                I have, over the years developed responsive creative web and mobile applications in areas such as
    ecommerce, smart systems, B2B and
    many more
              </p>
            </div>
          </div>
        </div>
        <div
          class="col-md-6 col-lg-4 my-3"
          data-aos="fade-right"
          data-aos-delay="200"
          data-aos-duration="1200"
        >
          <div
            class="feature-box-01"
          >
            <div
              class="icon"
            >
              <i
                class="icon icon-desktop"
              />
            </div>
            <div
              class="feature-content"
            >
              <h5>
                DevOps & Cloud Consulting
              </h5>
              <p>
                I provide cloud consultting, helping small business scale their systems to a wider audience based on
    their budgets, setting up ci/cd , kubernetes, docker & microservices architecture
              </p>
            </div>
          </div>
        </div>
        <div
          class="col-md-6 col-lg-4 my-3"
          data-aos="fade-right"
          data-aos-delay="400"
          data-aos-duration="1200"
        >
          <div
            class="feature-box-01"
          >
            <div
              class="icon"
            >
              <i
                class="icon icon-target"
              />
            </div>
            <div
              class="feature-content"
            >
              <h5>
                Third-Party Integration
              </h5>
              <p>
                Equipped with integrating a wide range of third-party libraries to help facilitate ease of security and
    optimal system reliability, such as payments gateways, SMEs and countless other services. 
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="separated"
        style="background-image: url(img/border-dark.png);"
      />
      <div
        class="title"
      >
        <h3>
          Awards.
        </h3>
      </div>
      <div
        class="row"
      >
        <div
          class="col-lg-4 m-15px-tb"
          data-aos="fade-right"
          data-aos-delay="0"
          data-aos-duration="1200"
        >
          <div
            class="feature-box-02 d-flex align-items-center"
          >
            <div
              class="icon"
            >
              <img
                alt="Kwasu nacoss Logo"
                src="img/award/a1.png"
              />
            </div>
            <div
              class="media-body"
            >
              <h6>
                KWASU NACOSS
              </h6>
              <p>
                Tech person of the year
              </p>
            </div>
          </div>
        </div>
        <div
          class="col-lg-4 m-15px-tb"
          data-aos="fade-right"
          data-aos-delay="200"
          data-aos-duration="1200"
        >
          <div
            class="feature-box-02 d-flex align-items-center"
          >
            <div
              class="icon"
            >
              <img
                alt="Global Entrepreneur Network Logo"
                src="img/award/a2.png"
              />
            </div>
            <div
              class="media-body"
            >
              <h6>
                GE Network (KWASU)
              </h6>
              <p>
                Winner of 2018 Global Entrepreneur Week
              </p>
            </div>
          </div>
        </div>
        <div
          class="col-lg-4 m-15px-tb"
          data-aos="fade-right"
          data-aos-delay="400"
          data-aos-duration="1200"
        >
          <div
            class="feature-box-02 d-flex align-items-center"
          >
            <div
              class="icon"
            >
              <img
                alt="Linux Foundation Training Logo"
                src="img/award/a3.png"
              />
            </div>
            <div
              class="media-body"
            >
              <h6>
                Linux Foundation
              </h6>
              <p>
                Scholarship Receipients
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="separated"
        style="background-image: url(img/border-dark.png);"
      />
      <div
        class="title"
      >
        <h3>
          Testimonials.
        </h3>
      </div>
      <div
        class="testimonial-wrapper"
      >
        <div
          class="slick-slider slick-initialized"
          dir="ltr"
        >
          <button
            class="slick-arrow slick-prev"
            data-role="none"
            style="display: block;"
            type="button"
          >
             Previous
          </button>
          <div
            class="slick-list"
          >
            <div
              class="slick-track"
              style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
            >
              <div
                aria-hidden="true"
                class="slick-slide slick-cloned"
                data-index="-1"
                style="width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    data-aos="fade-up"
                    data-aos-delay="400"
                    data-aos-duration="1200"
                    style="width: 100%; display: inline-block;"
                    tabindex="-1"
                  >
                    <div
                      class="testimonial-01 media"
                    >
                      <div
                        class="avatar"
                      >
                        <img
                          alt="review comments"
                          src="img/testimonial/team-3.jpg"
                        />
                      </div>
                      <div
                        class="media-body"
                      >
                        <p>
                          Ahmed communicates very clearly and does it with a big smile on his face. Since before graduating he’s worked on multiple projects for clients. Ahmed is always ready to pick a new, interesting project. He quickly takes ownership of projects as an individual contributor or on a leadership role being a force multiplier.
                        </p>
                        <h6>
                          Telent Team
                        </h6>
                        <span>
                          Talent Team at Gun.io
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="false"
                class="slick-slide slick-active slick-current"
                data-index="0"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    data-aos="fade-up"
                    data-aos-delay=""
                    data-aos-duration="1200"
                    style="width: 100%; display: inline-block;"
                    tabindex="-1"
                  >
                    <div
                      class="testimonial-01 media"
                    >
                      <div
                        class="avatar"
                      >
                        <img
                          alt="review comments"
                          src="img/testimonial/team-1.jpg"
                        />
                      </div>
                      <div
                        class="media-body"
                      >
                        <p>
                          Ahmed joined the team and quickly built out a high performing team across all levels of experience. He is extremely proactive, communicates effectively and in a very timely fashion. The quality and speed of his work is exceptional. But most of all he's thinking three steps ahead anticipating what we need to meet critical client facing deadlines. I frankly wish all of my staff had the technical expertise and business acumen that Ahmed possesses. 
                        </p>
                        <h6>
                          Patrina Mack
                        </h6>
                        <span>
                          CEO at YouROK Corp.
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="1"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    data-aos="fade-up"
                    data-aos-delay="200"
                    data-aos-duration="1200"
                    style="width: 100%; display: inline-block;"
                    tabindex="-1"
                  >
                    <div
                      class="testimonial-01 media"
                    >
                      <div
                        class="avatar"
                      >
                        <img
                          alt="review comments"
                          src="img/testimonial/team-2.jpg"
                        />
                      </div>
                      <div
                        class="media-body"
                      >
                        <p>
                           Ahmed has helped the junior members of the team learn. He has helped improve our website performance and led the migration of our Wordpress sites feom Bluehost to a VPS on DigitalOcean. He is helping us to staff a new project to develop our intranet and will lead that’s project too! 
                        </p>
                        <h6>
                          Georginna Feyst
                        </h6>
                        <span>
                          CEO at RADProd
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide"
                data-index="2"
                style="outline: none; width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    data-aos="fade-up"
                    data-aos-delay="400"
                    data-aos-duration="1200"
                    style="width: 100%; display: inline-block;"
                    tabindex="-1"
                  >
                    <div
                      class="testimonial-01 media"
                    >
                      <div
                        class="avatar"
                      >
                        <img
                          alt="review comments"
                          src="img/testimonial/team-3.jpg"
                        />
                      </div>
                      <div
                        class="media-body"
                      >
                        <p>
                          Ahmed communicates very clearly and does it with a big smile on his face. Since before graduating he’s worked on multiple projects for clients. Ahmed is always ready to pick a new, interesting project. He quickly takes ownership of projects as an individual contributor or on a leadership role being a force multiplier.
                        </p>
                        <h6>
                          Telent Team
                        </h6>
                        <span>
                          Talent Team at Gun.io
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide slick-cloned"
                data-index="3"
                style="width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    data-aos="fade-up"
                    data-aos-delay=""
                    data-aos-duration="1200"
                    style="width: 100%; display: inline-block;"
                    tabindex="-1"
                  >
                    <div
                      class="testimonial-01 media"
                    >
                      <div
                        class="avatar"
                      >
                        <img
                          alt="review comments"
                          src="img/testimonial/team-1.jpg"
                        />
                      </div>
                      <div
                        class="media-body"
                      >
                        <p>
                          Ahmed joined the team and quickly built out a high performing team across all levels of experience. He is extremely proactive, communicates effectively and in a very timely fashion. The quality and speed of his work is exceptional. But most of all he's thinking three steps ahead anticipating what we need to meet critical client facing deadlines. I frankly wish all of my staff had the technical expertise and business acumen that Ahmed possesses. 
                        </p>
                        <h6>
                          Patrina Mack
                        </h6>
                        <span>
                          CEO at YouROK Corp.
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide slick-cloned"
                data-index="4"
                style="width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    data-aos="fade-up"
                    data-aos-delay="200"
                    data-aos-duration="1200"
                    style="width: 100%; display: inline-block;"
                    tabindex="-1"
                  >
                    <div
                      class="testimonial-01 media"
                    >
                      <div
                        class="avatar"
                      >
                        <img
                          alt="review comments"
                          src="img/testimonial/team-2.jpg"
                        />
                      </div>
                      <div
                        class="media-body"
                      >
                        <p>
                           Ahmed has helped the junior members of the team learn. He has helped improve our website performance and led the migration of our Wordpress sites feom Bluehost to a VPS on DigitalOcean. He is helping us to staff a new project to develop our intranet and will lead that’s project too! 
                        </p>
                        <h6>
                          Georginna Feyst
                        </h6>
                        <span>
                          CEO at RADProd
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                aria-hidden="true"
                class="slick-slide slick-cloned"
                data-index="5"
                style="width: 0px;"
                tabindex="-1"
              >
                <div>
                  <div
                    data-aos="fade-up"
                    data-aos-delay="400"
                    data-aos-duration="1200"
                    style="width: 100%; display: inline-block;"
                    tabindex="-1"
                  >
                    <div
                      class="testimonial-01 media"
                    >
                      <div
                        class="avatar"
                      >
                        <img
                          alt="review comments"
                          src="img/testimonial/team-3.jpg"
                        />
                      </div>
                      <div
                        class="media-body"
                      >
                        <p>
                          Ahmed communicates very clearly and does it with a big smile on his face. Since before graduating he’s worked on multiple projects for clients. Ahmed is always ready to pick a new, interesting project. He quickly takes ownership of projects as an individual contributor or on a leadership role being a force multiplier.
                        </p>
                        <h6>
                          Telent Team
                        </h6>
                        <span>
                          Talent Team at Gun.io
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            class="slick-arrow slick-next"
            data-role="none"
            style="display: block;"
            type="button"
          >
             Next
          </button>
          <ul
            class="slick-dots"
            style="display: block;"
          >
            <li
              class="slick-active"
            >
              <button>
                1
              </button>
            </li>
            <li
              class=""
            >
              <button>
                2
              </button>
            </li>
            <li
              class=""
            >
              <button>
                3
              </button>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
